packages/ui/src/components/atoms/Radio/stories/Radio.stories.tsx

```diff
@@ -1,9 +1,8 @@
-import type { Story, Meta } from '@storybook/react'
+import type { Meta, Story } from '@storybook/react'
 import React from 'react'
 
+import { Radio as Component, RadioProps } from '@faststore/components'
```

Comment:
I think we can just remove this file for now, since we already have `Checkbox.stories.mdx`

Reply:
My bad. I've already removed it 🙈